src/components/PeoplePage/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,164 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../../types';
+import { SearchLink } from '../../SearchLink';
+import { PersonLink } from './PersonLink/PersonLink';
+
+type Props = {
+  visiblePeople: Person[],
+};
+
+export const PeopleTable: React.FC<Props>
+= ({ visiblePeople }) => {
+  const slugUrl = useParams();
+  const [sortType, setSortType] = useState('');
+  const [clickValue, setClickValue] = useState(0);
+
+  const isSelected = (personSlug: string) => {
+    return personSlug === slugUrl.slug;
+  };
+
+  const getParent = (parentName: string | null) => {
+    const parent = visiblePeople.find(
+      (person: Person) => person.name === parentName,
+    );
+
+    if (parent) {
+      return (
+        <PersonLink person={parent} />
+      );
+    }
+
+    return parentName || '-';
+  };
+
+  const sortByValue = (
+    value: string,
+    click: number,
+    setClick: (value: number) => void,
+  ) => {
+    setClick(click + 1);
+    switch (click) {
+      case 0:
+        setSortType(`${value}Asc`);
+        break;
+      case 1:
+        setSortType(`${value}Desc`);
+        break;
+      default:
+        setSortType('default');
+        setClick(0);
+        break;
+    }
+  };
+
+  const setSortedPeople = () => {
+    const sortedPeople = [...visiblePeople];
+
+    return sortedPeople.sort((a, b) => {
+      switch (sortType) {
+        case 'nameAsc':
```

Comment:
You can create a Enum for sortTypes in order not to hardcore them.
Also, you can have a sortType and sortOrder states to make your code a bit cleaner